judgels-backends/sandalphon/sandalphon-api/src/main/java/judgels/sandalphon/api/problem/bundle/ItemType.java

```diff
@@ -1,6 +1,9 @@
 package judgels.sandalphon.api.problem.bundle;
 
 public enum ItemType {
-    STATEMENT,
-    MULTIPLE_CHOICE
+    STATEMENT, MULTIPLE_CHOICE;
+
+    // String constants for use in annotations (evaluated at compile-time)
+    static final String Statement = "STATEMENT";
```

Comment:
This violates naming convention for constants (all uppercase).

I suggest to just hardcode the constants in `@JsonSubTypes`. It's clear enough and we only have 2 anyway.

Reply:
Changed